docs/cudf/source/basics.rst

```diff
@@ -0,0 +1,19 @@
+Basics
+======
+
+
+Supported Dtypes
+----------------
+
+cuDF uses dtypes for Series or individual columns of a DataFrame. cuDF uses NumPy dtypes, NumPy provides support for ``float``, ``int``, ``bool``,
+``'timedelta64[s]'``, ``'timedelta64[ms]'``, ``'timedelta64[us]'``, ``'timedelta64[ns]'``, ``'datetime64[s]'``, ``'datetime64[ms]'``,
+``'datetime64[us]'``, ``'datetime64[ns]'`` (note that NumPy does not support timezone-aware datetimes).
+
+
+The following table lists all of cudf types. For methods requiring dtype arguments, strings can be specified as indicated. See the respective documentation sections for more on each type.
+
+.. csv-table:: Supported dtypes in cudf
+   :file: dtypes.csv
+   :align: center
+   :widths: 30, 5, 30, 40, 50
+   :header-rows: 1
```

Comment:
Instead of making this CSV driven is there a way to build the table using markdown / rst formatting instead? It would make updating / maintaining it much smoother.

Reply:
Apart from [csv table](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#csv-files), there two different [ways](https://tables-with-sphinx.readthedocs.io/en/latest/index.html)

1. [List table](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#list-table-directive) Everything is vertically stacked so it would be tougher to maintain/edit - Probably more error-prone.
2. [Grid table](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#grid-tables) This would be a one-time effort of when creating and making some major restructuring and easier to make minor edits.

Reply:
I'm a big +1 to Grid Table

Reply:
I'll try out the Grid table for our setting and will go ahead with the grid table if there are no issues with it.